pkg/controller/manager/manager_controller.go

```diff
@@ -447,6 +447,20 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 	var tunnelSecret certificatemanagement.KeyPairInterface
 	var internalTrafficSecret certificatemanagement.KeyPairInterface
 	var linseedVoltronSecret certificatemanagement.KeyPairInterface
+
+	// We expect that the secret that holds the certificates for internal communication within the management
```

Comment:
We shouldn't need this at all in managed clusters - don't we need to put this within a branch like this?

```
if !managed {
}
```

?

Reply:
Manager is not installed on the managed clusters, only standalone and management.

Reply:
It is created by the kube-controllers controller. Historically that has always been the case IIRC.
See https://github.com/tigera/operator/blob/master/pkg/controller/logstorage/eskubecontrollers.go#L59

Reply:
Strange. Why is this certificate different from the others provisioned by the operator?

Reply:
> It is created by the kube-controllers controller

AH, I see - this is very different from "created by the KubeControllers" - which to my implies it is created by the kube-controllers pod itself. 

We should clarify this comment to make this more accurate.

Reply:
> Strange. Why is this certificate different from the others provisioned by the operator?

I don't quite remember. I thought @asincu was actually the person who first introduced this some years ago. My recollection of it, was that the kube controllers is passing the secret down the tunnel, but that would still not explain why it should be created here.

Reply:
> . I think the logic here was to have it created in kube-controllers because it is the first component that needs it (for MCM) and then propagate this to the rest of the components.

kube-controllers will only need this if it is talking to the manager pod though. This should be created and owned by the manager, and then kube-controllers can watch for it and update its bundle as needed if and when the secret is eventually created.